### PR TITLE
feat(settings): alternate idle tray icon

### LINKED ISSUE #1463: black tray icon is barely visible in gnome
### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### 📝 Description

When there are active notifications the icon becomes green and its clearly visible, by when not - it is black and I can't even tell if the app is running or not.

Maybe make default icon white on dark themes. Or add white background or shadow below the icon.

### 🪜 Steps To Reproduce

Start the app on any recent Ubuntu with Gnome and dark theme.

### Gitify Version

5.12.1

### Operating System

Linux

### GitHub Account

GitHub Cloud

### 📸 Screenshots

image

### PR DESCRIPTION
**Pull request by setchy**
fixes #1463 

Add new user controlled **System Setting** to control the color of the idle icon used within the menubar/tray

Default color is a black Gitify logo.  This user setting allows for a white Gitify logo.  Useful for devices which have a dark taskbar such as the latest Ubuntu release

### CODE REVIEW

File: src/electron/main.js [hunk 1]
```diff
@@ -189,9 +195,17 @@ app.whenReady().then(async () => {
...
   ipc.on('gitify:icon-idle', () => {
     if (!mb.tray.isDestroyed()) {
-      mb.tray.setImage(
-        updateAvailableMenuItem.visible ? idleUpdateAvailableIcon : idleIcon,
-      );
+      if (process.platform === 'darwin' || process.platform === 'win32') {
+        mb.tray.setImage(
+          updateAvailableMenuItem.visible ? idleUpdateAvailableIcon : idleIcon,
+        );
+      } else {
```
Comment (setchy) [inline src/electron/main.js:202]: Not 100% certain that all linux distros should use the white icon set 🤷‍♂️

Comment (setchy) [reply src/electron/main.js:202]: This _could_ be a user controlled setting, which would be more versatile, but adds some complexity to the impl

File: src/electron/main.js [hunk 2]
```diff
@@ -189,9 +195,17 @@ app.whenReady().then(async () => {
 
   ipc.on('gitify:icon-idle', () => {
     if (!mb.tray.isDestroyed()) {
-      mb.tray.setImage(
-        updateAvailableMenuItem.visible ? idleUpdateAvailableIcon : idleIcon,
-      );
+      if (process.platform === 'linux') {
```
Comment (setchy) [inline src/electron/main.js:198]: Not 100% certain that all linux distros should use the white icon set 🤷‍♂️

Comment (bmulholland) [reply src/electron/main.js:198]: Yeah isn't it more about dark vs light mode than linux vs other OS's?

Comment (setchy) [reply src/electron/main.js:198]: I'll update this PR to make it a user-controlled system setting. That way it's more versatile and agnostic of os/platform

Comment (setchy) [reply src/electron/main.js]: Updated logic pushed to use a user controlled system setting

**Review by bmulholland — APPROVED** [review]

**Pull request merged**